### ISSUE TITLE
(Suggestion) For the Cobble Gen (Vanilla, and mod compatible)

### ISSUE THREAD

**Issue by Thunder-Nova** [Mowmaster/Pedestals#35]
I just noticed this recently on a modded skyblock pack and felt that some things would be lovely
![image](https://user-images.githubusercontent.com/31967337/103468307-baeaa280-4d25-11eb-9beb-e5d60dcaed6d.png)

And saw that there are some common vanilla blocks that aren't seen, and would like to see if possible to add those.
- Diorite
- Andesite
- Granite

Many mods also tend to add blocks such as:
- Marble (Mods: Astral Sorcery, Environmental Materials)
- Limestone (Throughout the years seen this quite often) (Current Mod I know: Create)

I would add more blocks but there are too many and some not as common as the ones mentioned above. Modpack makers should add the rest in my opinion.

**EDIT:**
Would it be possible to make a simplify tag? Such as Pedestal:Commonblock and Pedestal:SilkyCommonblock
Which could use the block itself to make more.

**Comment by Mowmaster** [comment]
All the cobble gen 'blocks underneath' and outputs are able to be changed via datapacks. So if you need to customize it, the ability to do so already exists.
I didn't add support for diorite/andesite/granite because technically to get those you have to mine, or craft them with nether quartz.
There is a few mods that allow you to make diamonds from these blocks so I didn't want to support them by default.

Basically, Ive added support for a few things, its up to pack makers to add more or balance it themselves.

**Comment by Thunder-Nova** [comment]
> There is a few mods that allow you to make diamonds from these blocks so I didn't want to support them by default.

... wait what?

Then cobble would've been not allowed even by that standard. Since Ex Nihilo exist even now in 1.16.4
![image](https://user-images.githubusercontent.com/31967337/103480503-d34cd280-4da2-11eb-8098-fd88f26605d0.png)
